modules/cli/pages/actions-anonymization.adoc

```diff
@@ -1,34 +1,59 @@
 = Actions types for anonymization
 :description: Description of all the possible actions type for anonymization
+:javase-javadoc-base-url: https://docs.oracle.com/en/java/javase/21/docs/api
 
 == Overview
-Action types are mechanisms used to anonymize specific values in the exported data. A value can have one or multiple actions associated with it. If no actions are explicitly specified in the configuration for a particular value, it will default to a predefined action as described in the page xref:configuration-for-anonymization.adoc[Configure the anonymization].
+Action types are mechanisms used to anonymize specific values in the exported data. A value can have one or multiple actions associated with it.
+If no actions are explicitly specified in the configuration for a particular value, it will default to a predefined action as described in the page
+xref:configuration-for-anonymization.adoc[Configure the anonymization].
 
-Additionally, you can define a fallback mechanism in case the primary action does not work or does not find a match (for example, using the `regex_replace` action).
+Additionally, you can define a fallback mechanism in case the primary action does not work or does not find a match (for example, using the `REGEX_REPLACE` action).
 
-== Hash - Hash a value
+=== Order of execution of the actions
```

Comment:
**thought:** we should also mention the special case that occurs with `REPLACE_WITH_OTHER`

Reply:
Not finally there is not particular things for replace_with_other, the order wil be same as defined in configuration file.

Reply:
replace_with_other are applied in the end. Documented now.